src/pq/conninfo.cr

```diff
@@ -32,6 +34,8 @@ module PQ
     # The sslrootcert. Optional.
     getter sslrootcert : String?
 
+    getter auth_methods : Array(String) = ["scram-sha-256", "md5"] of String
```

Comment:
Should we maybe use symbols or some other not-string type here? It probably doesn't make huge difference, but it feels like using strings in crystal when you're not manipulating them is something Asterite would frown at.

Reply:
I know it'll come in as strings on the url parsing side, but maybe not allocating these 2 strings every time adds up. Not sure

Reply:
No, strings are totally the best choice here. We need to match against the configuration data which is... a string. So if this were symbols, they would need to be converted to string in order to compare it to a string. 0 benefit from symbols.
Actually, Asterite has repeatedly said that he would rather not have symbols in the language. So has Matz btw. regarding Ruby ;) Some people like symbols, but i think it's mostly a style thing. My wish for Crystal 2.0 (or later) is to remove them 😆

Reply:
Alright, makes sense :)